modin/experimental/cloud/rpyc_patches.py

```diff
@@ -0,0 +1,25 @@
+import numpy
+from rpyc.core import netref
+
+
+def apply_pathes():
+    def fixed_make_method(name, doc, orig=netref._make_method):
+        if name == "__array__":
+
+            def __array__(self, dtype=None):
+                # Note that protocol=-1 will only work between python
+                # interpreters of the same version.
+                res = netref.pickle.loads(
+                    netref.syncreq(self, netref.consts.HANDLE_PICKLE, -1)
```

Comment:
```suggestion
                    netref.syncreq(self, netref.consts.HANDLE_PICKLE, pickle.HIGHEST_PROTOCOL)
```
let's be explicit in protocol versioning to remove this uncertainty about Python versions' compatibility